apps/squareone/Dockerfile

```diff
@@ -36,6 +36,9 @@ COPY turbo.json turbo.json
 # ARG TURBO_TOKEN
 # ENV TURBO_TOKEN=$TURBO_TOKEN
 
+ARG SENTRY_AUTH_TOKEN
```

Comment:
Gah! [PR to address this](https://github.com/lsst-sqre/build-and-push-to-ghcr/pull/12)

Reply:
The NextJS Sentry integration includes a Webpack plugin that uploads sourcemaps to Sentry during the build process, which in our case, happens during the docker image build.